pipelines/match_injected_tract_catalog.yaml

```diff
@@ -0,0 +1,34 @@
+description: Match tract level injected catalog and input catalog.
+tasks:
+  matchObjectToTruth:
```

Comment:
These labels are camelCase, but everything else in `source_injection` is `snake_case`.

Reply:
Both `matchObjectToTruth` and `compareObjectToTruth` haven't been mentioned anywhere in the docs. Please make a reference to them explicitly somewhere, explaining what they do.